Add i18n Support (#650)
vapor/docs: README.md

# Documentation

Read the docs at [docs.vapor.codes](https://docs.vapor.codes)

## Contributing

> ### *Mistakes*

Reviewer: Lets remove this headline altogether

---

> If you have found any mistakes in the documentation, feel free to create a PR to fix it.

Reviewer: ```suggestion
If you want to add information to the docs or have found any mistakes you wish to fix, feel free to create a PR  for the change.
```

### *Translating*
---

> > For now this only applies to the Vapor 4 Documentation.

Reviewer: Can remove this since there will be no other documentation soon

> If you wish to contribute to the Vapor documentation by translating, follow these steps below.

Reviewer: ```suggestion
Localised docs are incredibly useful for allowing people to learn Vapor in their native language. If you wish to contribute to the Vapor documentation by translating, follow the steps below.
```

> To start of, install [Python3](https://www.python.org/download/releases/3.0/). Once you have done that, run `pip install -r requirements.txt` in the root directory of this repository. This will install all the needed dependencies in order to be able to build the documentation.  

Reviewer: ```suggestion
You'll need Python3 to build the docs. You can download this from the [Python Website](https://www.python.org/download/releases/3.0/) or install via Homebrew. Once installed, run `pip install -r requirements.txt` in the root directory of this repository. This will install all the needed dependencies in order to be able to build the documentation.  
```

Following the installation of the dependencies, check if your language you want to translate to is already included in the `mkdocs.yml` file. If it is not, then you can add it like this:
```yaml
languages:
  # Structure
  <language iso code>:
    name: <The name of the language>
    site_name: <The translated site name>

>     build: true # Whether the documentation gets build or not. Turn this to false for all languages you're not translating if building takes too long

Reviewer: ```suggestion
    build: true # Whether the documentation gets build or not. You can disable this if you don't want to build your language or want to temporarily disable other languages
```

  # Example
  nl:
    name: Nederlands
    site_name: Vapor Documentatie

>     build: false

Reviewer: Let's enable it by default to avoid confusing people!
```suggestion
    build: true
```

```

> > NOTE: The language iso code you have to insert has to conform to the ISO 639-1 Standard. More information can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).

Reviewer: Nit
```suggestion
> NOTE: The language code you have to add must conform to the ISO 639-1 Standard. More information can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
```
